maquetado/index.html

```diff
@@ -8,20 +8,127 @@
     <link rel="stylesheet" type="text/css" href="application.css" />
   </head>
   <body>
-      <main class="main-project">
-          <section class="section-project">
-            <h1 class="main-title m-bottom-1">Front-End Tools</h1>
-            <div class="tech-box">
-                  <article class="tech">
-                      <img src="../assets/react_icon.svg" alt="" class="tech-logo">
-                      <div class="tech-info">
-                          <h3 class="m-bottom-1">ReactJS</h3>
-                          <span class="tech-release m-bottom-1">Release: 2013</span>
-                          <p class="tech-description">React is a declarative, component-based JavaScript libar for building user interfaces</p>
-                      </div>
-                  </article>
-            </div>
-          </section>
-      </main>
+    <main class="main-project">
+      <section class="section-project l-container">
+        <h1 class="main-title m-bottom-1">Front-End Tools</h1>
+        <div class="techs-container">
+          <div class="row">
```

Comment:
Is this div really necessary? Could not be this styles applied to techs-container class instead?